app/Schedule/View.elm

```diff
@@ -269,6 +274,16 @@ predictionColor minutesLate =
             Color.red
 
 
+arrival : Date -> Train -> Node Msg
+arrival date train =
+    trainMetadata ("arrives at" ++ arrivalTime train)
```

Comment:
does `arrivalTime` prepend whitespace to the string? The lack of it after `"arrives at"` caught me a bit off guard